src/main/plugins/calculator-plugin/calculator-plugin.ts

```diff
@@ -53,4 +53,10 @@ export class CalculatorPlugin implements ExecutionPlugin {
             resolve();
         });
     }
+
+    thousands_separators(num: string) {
+        const numParts = num.toString().split(".");
+        numParts[0] = numParts[0].replace(/\B(?=(\d{3})+(?!\d))/g, this.config.separator);
+        return numParts.join(".");
+    }
```

Comment:
This function name does not match this project's coding style. Additionally I think that `num` is not an optimal variable name, because it does not really describe what it is.